tests/test_spline_interpolation.cpp

```diff
@@ -459,8 +558,46 @@ TEST_F(SplineInterpolationTest, quintic_spline_with_end_point_velocity)
     ASSERT_TRUE(g_ur_driver_->writeTrajectoryControlMessage(urcl::control::TrajectoryControlMessage::TRAJECTORY_NOOP));
 
     // Ensure that we follow the joint trajectory
-    urcl::vector6d_t expected_joint_positions;
+    urcl::vector6d_t expected_joint_positions, change_acc;
     interpolate(spline_travel_time, expected_joint_positions, coefficients);
+
+    if (init_acc_test && loop_count == 0)
+    {
+      last_joint_acc = joint_acc;
+    }
+    else if (init_acc_test && last_joint_acc != joint_acc)
+    {
+      init_acc_test = false;
+      loop_count = 1;
+      last_joint_acc = joint_acc;
+      last_change_acc.fill(0.0);
+    }
+
+    if (loop_count % REDUSE_FACTOR == 0)
+    {
+      last_joint_acc = joint_acc;
+      last_change_acc.fill(0.0);
+    }
+    else
+    {
+      for (unsigned int i = 0; i < last_joint_acc.size(); ++i)
+      {
+        change_acc[i] = joint_acc[i] - last_joint_acc[i];
+
+        if (!nearly_equal(change_acc[i], 0.0, EPS_ACC_CHANGE) && !nearly_equal(last_change_acc[i], 0.0, EPS_ACC_CHANGE))
+        {
+          // Acceleration should only increase or be constant within one scaled timescale.
+          // It should not fluctuate to zero or overshoot
+          EXPECT_EQ(sign(last_change_acc[i]), sign(change_acc[i]))
+              << " acceleration change direction doing "
```

Comment:
I think a debug message wasn't fully deleted here.

Reply:
This is intended. The debug message is only printed out if the EXPECT_EQ fails.